nevergrad/benchmark/experiments.py

```diff
@@ -77,6 +100,21 @@ def naivemltuning(seed: tp.Optional[int] = None) -> tp.Iterator[Experiment]:
     for xp in internal_generator:
         yield xp
 
+# We register only the sequential counterparts for the moment.
+@registry.register
+def seqkerastuning(seed: tp.Optional[int] = None) -> tp.Iterator[Experiment]:
+    """Sequential counterpart of mltuning."""
+    internal_generator = kerastuning(seed, overfitter=False, seq=True)
+    for xp in internal_generator:
+        yield xp
+
```

Comment:
A comment I don't usually make because it's not very important, but given you help a lot in the maintenance I'll make it so that you are aware of it:
Top-level functions and classes should be surrounded by 2 blank lines for pep8 styling (which we as much as possible follow in nevergrad):
https://www.python.org/dev/peps/pep-0008/#blank-lines